experimental/PropertyDDS/packages/property-inspector-table/package.json

```diff
@@ -55,10 +59,6 @@
     "@fluid-experimental/property-properties": "^0.58.1000",
     "@fluid-experimental/property-proxy": "^0.58.1000",
     "@fluidframework/build-common": "^0.23.0",
-    "@hig/fonts": "^1.0.2",
-    "@material-ui/core": "^4.0.0",
-    "@material-ui/lab": "^4.0.0-alpha.16",
-    "@material-ui/styles": "^4.1.1",
```

Comment:
@ruiterr @karlbom @nedalhy @evaliyev @microsoft/fluid-cr-dds These were all duplicated in the deps section. I assume that was unintentional, but let me know if you know of a reason for the dupes.

Reply:
this sort of situation should probably be considered an error when validating the package.json. for fluid packages, the tools miss this case when updating dependencies then later get upset that versions don't match

Reply:
Thanks @tylerbutler for cleaning it up , yes it was probably unintentional.